enterprise/server/backends/migration_cache/migration_cache.go

```diff
@@ -834,20 +836,26 @@ func (mc *MigrationCache) copyDataInBackground() error {
 	}
 }
 
-func (mc *MigrationCache) logCopyChanFullInBackground() {
-	ticker := time.NewTicker(mc.copyChanFullWarningInterval)
-	defer ticker.Stop()
+func (mc *MigrationCache) logCopyChanInfoInBackground() {
+	copyChanFullTicker := time.NewTicker(mc.copyChanFullWarningInterval)
+	defer copyChanFullTicker.Stop()
+
+	metricTicker := time.NewTicker(5 * time.Minute)
```

Comment:
There's no harm in making this much finer, say every 5 seconds. 5 minutes might be enough time to burst and drain a lot of copies from the queue which would not really give you a good insight into the behavior of the queue.

Reply:
Ack